olive/passes/onnx/mixed_precision.py

```diff
@@ -0,0 +1,145 @@
+import logging
+from typing import Any, Dict, List
+
+from olive.model import ONNXModel
+from olive.passes import Pass
+from olive.passes.onnx.common import get_external_data_config, model_proto_to_olive_model
+from olive.passes.pass_config import PassConfigParam
+
+logger = logging.getLogger(__name__)
+
+
+class OrtMixedPrecision(Pass):
+    """Convert model to mixed precision."""
+
+    _requires_user_script = True
+
+    @staticmethod
+    def _default_config() -> Dict[str, PassConfigParam]:
+        config = {
+            "op_block_list": PassConfigParam(
+                type_=List[str],
+                default_value=["SimplifiedLayerNormalization", "SkipSimplifiedLayerNormalization", "Relu", "Add"],
```

Comment:
The default value for gpt2 is different. For the gpt2 case, do we expect the user config to pass the correct one? https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/python/tools/transformers/models/gpt2/gpt2_helper.py#L549

Reply:
I’m using the default value for whisper model: https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/python/tools/transformers/models/whisper/whisper_helper.py#L152. I remember that we had a discussion before that Olive can support different common models, maybe when we have different model cases like whisper gpt2 hf we can have different default values for them.

Reply:
Sounds good to me! We can worry about the other models when it comes up. 

I saw somewhere where the default is something like "auto". if "auto", then we can look at the model type to create the default list. But it is not needed for now.